cmd/govim/buffer_events.go

```diff
@@ -130,6 +131,10 @@ func (v *vimstate) bufUnload(args ...json.RawMessage) error {
 }
 
 func (v *vimstate) handleBufferEvent(b *types.Buffer) error {
+	if err := v.redefineHighlights(v.diagnostics()); err != nil {
+		v.Logf("failed to update highlights for buffer %d: %v", b.Num, err)
```

Comment:
In a later PR I think we perhaps need to show the user an error popup in addition to logging.

Which means it might well be time therefore to extract something common from:

https://github.com/govim/govim/blob/3376e910e567b9a0e09994f0f0f5cb6c82f3f5f8/cmd/govim/gopls_client.go#L28-L57

Reply:
Indeed, wrote #641 as reminder